.drone.yml

```diff
@@ -1,44 +1,45 @@
-# After any change to this file you MUST regenerate and checkin
-# a .drone.sec even if no secrets were changed. The secrets file is tied
-# to a specific .drone.yml so artifacts will not be uploaded to (OR DOWNLOADED
-# FROM) GCP in following builds without an updated .drone.sec
-#
-# When you are ready to submit a pull request, you must regenerate .drone.sec for the vmware/vic repo:
-#
-# $ drone -s https://ci.vmware.run -t ${DRONE_TOKEN} secure --in "secrets.yml" --out ".drone.sec" --repo vmware/vic --yaml ".drone.yml"
-#
+# After any change to this file you MUST re-sign and checkin the .drone.yml.sig
+# When you are ready to submit a pull request, you must regenerate .drone.yml.sig for the vmware/vic repo:
+# $ drone sign -s https://ci.vcna.io -t ${DRONE_TOKEN} vmware/vic
```

Comment:
I thought drone 0.5 was supposed to manage this in a db now instead of our internal secrets yaml?